lib/cs_guide_web/controllers/csv_controller.ex

```diff
@@ -79,6 +81,35 @@ defmodule CsGuideWeb.CsvController do
      }
   end
 
+# Venues and Drinks
+  defp venues_drinks_csv_content(conn) do
+    venues_drinks = Venue.all()
+    |> Venue.preload([:drinks])
+    |> Enum.map(&venues_drinks_csv_data(&1))
+    |> Enum.flat_map(&(&1))
```

Comment:
`&(&1)` it the identity function in elixir, ie the function return its parameter without any changes on it.
We have to call this to flatten the list of nested drinks